.azure-pipelines/testscripts_analyse/report_data_storage.py

```diff
@@ -0,0 +1,92 @@
+"""Wrappers and utilities for storing test reports."""
+import json
+import os
+import tempfile
+
+from azure.kusto.data import KustoConnectionStringBuilder
+
+try:
+    from azure.kusto.ingest import KustoIngestClient
+except ImportError:
+    from azure.kusto.ingest import QueuedIngestClient as KustoIngestClient
```

Comment:
This workaround is for compatibility issue. If you are using latest lib, no need to run the code like this.